docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -1,22 +1,30 @@
 {
```

Comment:
<blockquote>Finally, we utilize the functionality in our <code>submit_qubo()</code> function to solve the optimization problem of our objective function as a QUBO using multiple techniques. To employ  the simulated annealing technique, we can add a <code>method="dry-run"</code> parameter</blockquote><p>nit: We don't invoke <code>submit_qubo</code> until two cells later, so IMO this text referring to it should only come after that.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/558/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>done!</p>